packages/admin/cms-admin/src/contentScope/Provider.tsx

```diff
@@ -1,16 +1,24 @@
 import { createContext, Dispatch, ReactNode, SetStateAction, useCallback, useContext, useMemo, useState } from "react";
-import { generatePath, match, Redirect, Route, Switch, useHistory, useRouteMatch } from "react-router";
+import { match, Redirect, Route, Switch, useHistory, useRouteMatch } from "react-router";
 
 export interface ContentScopeInterface {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     [key: string]: any;
 }
 
+type ContentScopeLocation<S extends ContentScopeInterface = ContentScopeInterface> = {
+    createPath: (scope: ContentScopeValues<S>) => string | string[];
```

Comment:
This is the noteworthy change: Adding support for string arrays here allows us to use multiple paths in the provider. This is what react-router does in their docs as well: https://v5.reactrouter.com/web/api/Route/path-string-string